bundle/deploy/terraform/init.go

```diff
@@ -37,7 +38,7 @@ func (m *initialize) findExecPath(ctx context.Context, b *bundle.Bundle, tf *con
 		return tf.ExecPath, nil
 	}
 
-	binDir, err := b.CacheDir("bin")
+	binDir, err := b.CacheDir(context.Background(), "bin")
```

Comment:
Any reason for this to be context.Background() and not ctx?

Reply:
No, it should be ctx.

This hasn't been an issue because `b.CacheDir` only accesses an env var for temporary directories, and that isn't overwritten for a specialized test for this function. This can be updated to `ctx` without issues.

Reply:
https://github.com/databricks/cli/pull/2208